examples/reinforcement-learning/ddpg.py

```diff
@@ -0,0 +1,98 @@
+import dynet as dy
+import numpy as np
+
+from memory import Memory
+from network import MLP
+
+
+class DDPG:
```

Comment:
Could you add a comment with a reference that readers can reference to understand this method?